source/numerics/vector_tools_project.inst.in

```diff
@@ -67,3 +67,27 @@ for (VEC : SERIAL_VECTORS ; deal_II_dimension : DIMENSIONS; deal_II_space_dimens
     \}
 #endif
 }
+
+for (VEC: REAL_NONBLOCK_VECTORS; deal_II_dimension : DIMENSIONS; deg : DEGREES)
```

Comment:
No, the script doesn't allow for this right now. If you want, open an issue for this to be implemented.